src/bin/async-benchmark.rs

```diff
@@ -0,0 +1,28 @@
+use arrow_zarr::async_reader::{ZarrPath, ZarrRecordBatchStreamBuilderNonBlocking};
```

Comment:
nit, but missing copyright. DataFusion at least has a checker as part of its CI, which may make sense to use here.

Reply:
Added a copyright here and in several other files.